src/main/fc/controlrate_profile.c

```diff
@@ -40,14 +40,20 @@ void pgResetFn_controlRateProfiles(controlRateConfig_t *instance)
     for (int i = 0; i < MAX_CONTROL_RATE_PROFILE_COUNT; i++) {
         RESET_CONFIG(controlRateConfig_t, &instance[i],
             .rcExpo8 = 70,
+            .manual_pitch_rcExpo8 = 70,
```

Comment:
Do we really need separate expo for MANUAL?

Reply:
More like simplicity. If there is no need to have it, probably it's not needed.

Reply:
Maybe separate roll and pitch expo is not really needed. But I've to point those things:
1)Most pilot that use PASS mode use additional expo in their FrSky radios, I use FlySky and I cannot do it, will be nice if FC does it instead of the transmitter.
2)We really should find a way to link rcExpo8 with manual_rcExpo8. Like people increasing expo when in stabilized mode but ending not touching expo when in manual mode and having to change two settings is not very ideal, ok but not ideal :) What about doing like, manualRcExpo=100 in Unit %, as we do for the rates? 100 will be default and then one can increase or decrease expo in regards to rcExpo value.

Reply:
@giacomo892 
1) Not sure I understand. The changes already makes that possible (more expo in manual compared to the other modes). Are you asking to change the way I done it or only saying it is better now.
2) I'm not against this change

Reply:
@shellixyz on that point I was responding to @DzikuVx that suggested not to add an additional expo :)

Also I think we should drop the current PASSTHROUGH MODE for this new one. We may keep the current name too but with this new settings. There aren't reasons to keep two "manual" modes

Reply:
Ok :)

I created a separate mode because I think it is important to keep the `PASSTHROUGH` mode. It is the one allowing to find the right values for the roll/pitch/yaw FF parameters and also to check max control surfaces deflection. If we remove it you will need to make sure to set the `MANUAL` mode rates to 100% because you can check if the FF params are set correctly and then restore the `MANUAL` mode rates to their original values.

Reply:
But it is also true that by default the "new" PASSTHROUGH will behave 100% like the old one :)

Reply:
I agree with @giacomo892 on this. There are valid reasons for having both MANUAL and PT modes, but I think it will end being too confusing and we should try to have just one non stabilised mode.

A question I frequently ask myself when deciding wether to add/drop/how to organise features is: "how can I present this to the user so I can get away with minimal documentation?". Two "almost-the-same" but not "quite-the-same" modes is definitely a pain to document.

Also, MANUAL is a better name than PASSTHROUGH. The former will be instantly clear to any pilot, the latter... doesn't seem that obvious to me.

So my vote would be to rename PT to MANUAL and add these new settings.

Reply:
Right, I also agree. Simpler is always better. The user will only have to put the rates back to 100% to set/check the FF terms and not forget to put them back to the desired value. I will replace PT with the MANUAL mode.

Reply:
@shellixyz Just make the new settings as %. 100 will be the default so the users really don't need to do anything and if they change the rates/expo for stabilized modes they will apply to MANUAL too. This allow transparent diff restore and nothing will change for users that wants the old PASSTHROUGH behaviour. The documentation will be also very simple.

Reply:
What are we trying to achieve here people? Aeroplanes are not Quads, It's an autopilot not a FULL flight controller. Rates are not electronic they are mechanically chosen. Expo is not a control it's a personal "stick feel" thing.
For the types of plane we would use INAV on they should be setup in Passthrough with 0 expo starting point and if you can not get the desired  control, no autopilot will ever fly it correctly either !
only then we then go on to setup the Autopilot  etc..

Why would we want/need another set of electronic parameters in a "manual" mode that at best can only soften the existing direct/passthrough mode?
If you want to change the feel of any of the flight modes simply assign some Rate/expo in your radio.
It was not long ago most were still running the "Then" default 200% servo mixes and somehow getting a setup out of it, maybe if people start turning off the expo the planes won't feel like merde :) Unless you are flying high alpha with 85deg plus control deflections you don't use 70% expo - INAV  should default to zero when Aeroplane is selected

Reply:
It only adds the possibility to use different expo settings for manual control and for stabilized modes. And also the possibility to reduce the control surfaces deflection in manual control. You can do the same by not using expo on the FC but only if you have a programmable remote control. It is a very simple mod. When in-flight profile switching will be available it will also allow for high/low rates selection in manual control without remote programmation. @giacomo892 and I are both using a FS-i6 for some of our models you can't do much on these. I have been using this patch for some time because I need it and apparently I'm not the only one as the feature was requested.

Reply:
@shellixyz   Rate has nothing to do with control surface deflection you need to change the mix values for that. RC Rate scaler has disappeared for a Quad deg/sec Betaflight style.  Sounds to me like you are trying to get the right feel from the plane in different modes and playing with expo to cover up a bad setup. I would say I have not used the latest INAV on a Plane but if Angle is as Dead as on a Quad out of the box I understand your issue - I don't know as I say.:D

Regardless set the thing up as I say all in passthrough, CofG, lateral balance, motor thrust line, control deflection ( enough for full control but not enough to induce a spin if you bury the controls in normal flight) etc.. then finally add expo to your taste.
The Aeroplane is now setup and you tune the other modes to work with that perfect Mechanical setup and I am sure you will not require different expo and movement because it is not possible for the FC to do anything different with the controls in regards to increasing deflection - logical enough yes?
Then if the thing can not be made to fly nicely in say Angle and Acro we can get all the clever coders to find out why ;)  But I'm sure there is enough parameter options to tweak  the stabilized modes to not have to resort covering issues with RC rate and expo which it sounds like you are proposing.

Reply:
@shellixyz I was referring to servo rates (aka control surface deflection) too not gyro rates since fixing maximum gyro rates for a manual mode is not a big deal. (you are no more manual then).

@Redshifft nobody wants to hide untuned planes or hiding bad performance in stabilized modes. PASSTHROUGH must remain a mode where there is not stabilization at all. Only additional expo (or less) if needed and the possibility to change different servo rates to allow more or less surfaces deflection.

What do you think now?

Reply:
What do I think now?....Mamma mia ! :D

Reply:
@Redshifft Sometimes you want more precise control when flying unstabilized that's just one way to achieve it. And it's the only way for basic remotes. If you don't have the need for reduced servo travel unstabilized mode you don't have tu use it and by default it will behave just as before (PASSTHROUGH).

Reply:
@giacomo892 Yes for manual mode when I'm talking about rates I'm talking about servo travel in % nothing to do with the gyro in this case.

Reply:
@shellixyz Sorry! I misread the code. I think by renaming all to passthrough and cleaning up this will be great to use!

Reply:
@shellixyz Well I think we have too many modes and features already, and while I sympathise if people can't do basic stuff with their TX's, just remember many of us were forced to discard high quality Radios for cheap low end(in comparison)equipment to have functionality with all this stuff and that does still bug me as we have half destroyed the original RC market and innovators.

You two can both code why don't you do something between yourselves? But I really don't think we need it - It does not help the fact I'm sitting here with a little Afromini FC perfect for this small plane but I can't use it because Inav has got so bloated I can't load the latest release and it's a beautiful high quality made in Japan board :( yet it's still more than powerful enough to control a simple Aeroplane.

Reply:
@Redshifft I don't understand your issue. We are just adding config variables for manual rates in the rate profiles. Also we discussed it and the goal is to make the manual mode for the moment called PASSTHROUGH called like it should be called, that is MANUAL. We are not even adding a new flight mode in the end. Just extending the possibilities of the flight mode that was already there. When I was using the RX without an FC I used the high/low rates and still expect to be able to use it with a FC supposed to increase the possibilities not lower them.

I don't think syncing flight mode switch positions and high/low rates on the remote is a basic thing. I'm only using this radio because it is small and you are complaining we are killing the RC market innovation by using this radio but you on the other end are using FCs with a very old MCU and not be able to use the latest version of flight control software instead of driving the market forward by upgrading to a recent board with a decent MCU. It's like saying that people who buy cheap new computers fitting their needs are stalling the CPU developments and at the same time using a computer 15 years old and complaining you can't run Word 2048.

The source is open you can also build yourself a light version between yourself and the 2 others users of F1 MCUs left in this world. That's what I did when I was still using a Naze and couldn't flash a full version with all the things built-in and software configurable for maximum ease of use. F1 boards have been deprecated for a long time before the time arrived to ditch the support for *ready made builds* but you can still use it just make some effort and build yourself a lighter version it is very easy to reduce the build size by disabling the stuff you don't need also you are describing the recent versions bloated so I think you won't need much time to find features to disable and make it fit.

Reply:
@Redshifft You're going way off topic. Please, try to keep the discussion related to the PR and respect everyone.

I myself configure the expo/rates for PT mode in the radio, but that doesn't mean there's no value in having it in the FC. Some radios don't support it at all and having all model configuration centralised in the FC rather than split between radio and FC is definitely helpful too.

Reply:
@shellixyz  I've said my bit on this  but as I'm out-gunned on this one there is little point in me saying more than, this has never come up as an issue before and it's bound to be more work than a line of code to make it work :)

Reply:
@Redshifft To make what work? If you are talking about this PR it is already working fine. If you are talking about the F1 board no line of code should be required only undefining some defines.

Reply:
@shellixyz Thought you wanted separate rate/mixer and expo values for passthrough?

That F1 board is a rev3 Afromini but it never had a separate  target just the generic Naze, but it seems DE has pulled all the F1 stuff.

Reply:
I agree with the points @Redshifft has mentioned. Without any additional features in iNav it is already possible to set up and fly an aircraft in PASSTHROUGH and using RC EXPO in iNav (in PASSTHROUGH) to adjust stick feel. If the aircraft feels more / less agile in ACRO compared to PASSTHROUGH, it is just a matter of lowering / increasing the rates in ACRO. Adjusting an aircraft in let's say ACRO mode first and then trying to compensate any weird behavior in PASSTHROUGH by additional parameters doesn't sound logical to me.

Reply:
@Dronek The issue at least for me is that I can't use less control surfaces movements in passthrough compared to the other modes. Well I could but not without sacrificing channels or other functionality and even then it wouldn't be very practical or won't allow me to configure the mode switch(es) as I like.

@Redshifft I can add preprocessor directives to disable the new settings if you like. I personally don't think it is important because there are a lot other less basic things you can disable. Like Spektrum LED strip support (first things coming to my mind).

Reply:
@shellixyz that would be good :) I did go as far as loading most of the compiler and dependencies etc and I thought, I will need lots of other stuff etc and software is just not my thing! If I had everything in place I'm sure I could have got something half usable in about 2 weeks :D 
It's a simple little board with no USB bridge chip no SPI just 6050 and bmp280 on I2c, no flash.
Just need MSP the 2 ublox autocfgs PPM rx voltage and current adc's and the just the Airplane half of 1.8.1 (although I doubt that can be split easily) soft serial not required but gps and blheli passthrough would be very useful, Thanks